_tools/generate.js

```diff
@@ -1,53 +1,155 @@
 const fs = require("fs").promises;
 const path = require("path");
 
-const { save, getTemplate, insert } = require("./utils");
+const { save, getTemplate, insert, DefaultListMap } = require("./utils");
+const Handlebars = require("handlebars");
+
 
 start();
 
 async function start() {
+  await Promise.all([generate("norge"), generate("sverige")]);
+}
+
+async function generate(country) {
   try {
     const template = await getTemplate("index_template");
-    const documents = await getDocuments();
-    const documentItems = documents.map(createItem).join("\n");
+    const documents = await getDocuments(country);
+
+    const headerList = [
+      createCountryRadioGroup(country),
+      createCollapsibleLists(documents),
+    ];
 
     const result = insert(template, {
-      content: documentItems
+      content: headerList.join("\n"),
     });
 
-    save("index", result);
+    save(country === "norge" ? "index" : "sverige", result);
 
-    console.log("Generated index.html");
+    console.log(`Generated ${country}.html`);
   } catch (e) {
     console.error("Error:", e);
   }
 }
 
-function createItem(item) {
+function createCountryRadioGroup(country) {
+  const norgeActive = country === "norge" ? " active" : "";
+  const sverigeActive = country === "sverige" ? " active" : "";
+
   return `
-  <article>
-    <h3><a href="${item.url}" title="${item.title}">${item.title}</a></h3>
-  </article>
-`;
+    <div class="countryRadioGroup">
+      <label class="toggleOption toggleOptionLeft${norgeActive}">
+      Norge
+        <a href="/" class="countryRadioButton${norgeActive}" aria-current="${country === "norge" ? "page" : ""}" tabindex="0" aria-label="Klikk for å gå til norske avtaler"></a>
+      </label>
+
+      <label class="toggleOption toggleOptionRight${sverigeActive}">
+      Sverige
+        <a href="/sverige" class="countryRadioButton${sverigeActive}" aria-current="${country === "sverige" ? "page" : ""}" tabindex="0" aria-label="Klikk for å gå til svenske avtaler"></a>
```

Comment:
Skal det være href="/sverige.html" her? Finner ikke filen når jeg tester lokalt

<img width="411" alt="Screenshot 2023-05-19 at 13 02 43" src="https://github.com/varianter/dokument-mal/assets/110395932/eef98529-84d3-472d-8060-24ef00ccec90">

Reply:
Kun hvis man ikke har http server da, så er lite kritisk

Reply:
Hele Sverige greiene er nok midlertidig. Tror en mer permanent løsning ville vært å lagt svenske avtaler på 
{svensk ord for avtaler}.variant.se men dit kommer vi vel etter hvert.